scripts/index.html

```diff
@@ -34,14 +34,14 @@ <h1 id="page-title">
         <div style="display: flex; justify-content: space-between;">
           <div>
             <h2>Local Stream:</h2>
-            <audio id="local-audio" autoplay playsinline></audio>
+            <!-- <audio id="local-audio" autoplay playsinline></audio> -->
```

Comment:
Why don't you remove it, instead of commenting it out?
Or, we could mute it maybe? so we can still do testing for local audio